Common storage backend docs (#2347)
grafana/mimir: docs/sources/operators-guide/configuring/about-configurations.md

To see the current configuration state of any component, use the [`/config`]({{< relref "../reference-http-api/index.md#configuration" >}}) or [`/runtime_config`]({{< relref "../reference-http-api/index.md#runtime-configuration" >}}) HTTP API endpoint.

## Common configurations

> Some configurations, like object storage backend, are repeated for multiple components.

Reviewer: ```suggestion
Some configurations, such as object storage backend, are repeated for multiple components.
```

> To avoid the repetition in the configuration file, the [`common`]({{< relref "../configuring/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags can be used.

Reviewer: ```suggestion
To avoid the repetition in the configuration file, use the [`common`]({{< relref "../configuring/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags.
```

> Common configurations are applied first to all the specific configurations, allowing them to be overridden later by specific values.

Reviewer: ```suggestion
Common configurations are first applied to all the specific configurations, which allows  them to be overridden later by specific values.
```
The are not overwritten, but overridden, right? If so, leave as is.

Author: I think they are overridden (actually, _their values_ are overridden).

Reviewer: So there is no writing going on. :)

> For example, this configuration:

Reviewer: ```suggestion
For example, the following configuration uses the same Amazon S3 object storage bucket called `mimir`. The common storage is located in the `us-east` region for both ruler and alertmanager stores, and the blocks storage uses the `mimir-blocks` bucket from the same region:
```

> For example, this configuration:
> 
> ```yaml
> common:
>   storage:
>     backend: s3
>     s3:
>       region: us-east
>       bucket_name: mimir
> 
> blocks_storage:
>   s3:
>     bucket_name: mimir-blocks
> ```
> 
> Will use the same Amazon S3 object storage bucket called `mimir` and located in the `us-east` region for both ruler and alertmanager stores, while blocks storage will use the `mimir-blocks` bucket from the same region. See [object storage configuration reference ]({{< relref "configuring-object-storage-backend.md" >}}) for more detailed reference of this configuration.

Reviewer: ```suggestion
For example, the following configuration uses the same Amazon S3 object storage bucket called `mimir`. The common storage is located in the `us-east` region for both ruler and alertmanager stores, and the blocks storage uses the `mimir-blocks` bucket from the same region:

```yaml
common:
  storage:
    backend: s3
    s3:
      region: us-east
      bucket_name: mimir

blocks_storage:
  s3:
    bucket_name: mimir-blocks
```

For a reference of this configuration, see [object storage configuration reference ]({{< relref "configuring-object-storage-backend.md" >}}).
```
